particle/particle/particle.py

```diff
@@ -48,13 +48,102 @@ class Particle(object):
+
+        Note:
+        This name does not contain the charge. See alternative `fullname`.
+
+    pdgid
+        The PDG ID.
+
+    quarks
+        The quark content of the particle. Empty string if not relevant.
+
+        Note:
+        Capital letters represent anti-quarks, 'qQ' stands for light quark,
+        light anti-quark pair with unknown wave functions.
+        x, y stand for wave function coefficients, see the Review of Particle Physics (RPP) 'Quark Model' Review.
+        p, q stand for CP violation parameters, see the RPP 'CP violation in KL Decays' review.
+
+    rank
+        The particle rank as specified by the PDG, i.e. the number of baryon stars - used only on baryons.
+
+        Possible values are:
+        4 - Existence is certain, and properties are at least fairly well explored.
+        3 - Existence ranges from very likely to certain, but further confirmation
+             is desirable and/or quantum numbers, branching fractions, etc. are not well determined.
+        2 - Evidence of existence is only fair.
+        1 - Evidence of existence is poor.
+
+    status
+        The particle status as specified by the PDG.
+
+        Possible values are:
+        R - Established particles in the Review of Particle Physics (RPP) Summary Table
+            in Particle Physics Booklet
+            (established quarks, gauge bosons, leptons, mesons and baryons, except those in D below).
+        D - The particle is omitted from the Summary Tables in the Particle Physics Booklet,
+            but not from the Review. These entries are omitted only to save space
+            even though they are well established.
+        S - The particle is omitted from the particle properties Summary Tables
+            because it is not well established.
+        F - Special case: "Further mesons", see RPP, these states are in the RPP database
+            but are poorly established or observed by a single group and thus need confirmation.
+            If used, these should be referred to the original publication.
+
+    width
+        The particle decay width, in MeV.
+
+    width_lower
+        The lower uncertainty on the particle decay width, in MeV.
+
+    width_upper
+        The upper uncertainty on the particle decay width, in MeV.
+
+    anti_flag
+        The particle-antiparticle flag.
+
+        A = B     - particle that has anti-particle partner different from particle
+                    with ASCII name formed by concatenation of the name shown below with charge
+                    ( e- <--> e+, pi+ <--> pi-, K+ <--> K-, W+ <--> W- ).
+        A = F     - particle that has anti-particle partner different from particle
+                    with ascii name formed by concatenation of the name shown below with string "bar" and charge
+                    by the rule (nu(e) <--> nubar(e), p <--> pbar, Delta++ <--> Deltabar--)
+        A = blank - particle that coincide with its antiparticle (gamma, pi0, eta).
```

Comment:
Coincides

Reply:
Thanks - I copied this info from the extended PDG file and fixed a few typos on the way. This one slipped.